stable/percona-xtradb-cluster/templates/secrets.yaml

```diff
@@ -24,3 +24,24 @@ data:
   {{ else }}
   xtrabackup-password: {{ randAlphaNum 10 | b64enc | quote }}
   {{ end }}
+{{- if .Values.ssl.enabled }}
+{{ if .Values.ssl.certificates }}
+{{- range .Values.ssl.certificates }}
```

Comment:
I think we can remove this range

Reply:
Range is needed because the `.Values.ssl.certificates` is a list.

Reply:
That's fair, I mainly wanted to maintain parity with the `stable/mysql` chart so that they functioned the same way.  If need be, I can remove the list and change to a conditional.

Reply:
I didn't know this is kind of copy from stable/mysql.